### PR TITLE
4.3.0rc0

### PR DESCRIPTION
**Pull request by phoebusm**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by phoebusm** [comment]
@conda-forge-admin, please rerender

**Comment by jjerphan** [comment]
@conda-forge-admin, please rerender

**Review by jjerphan — APPROVED** [review]
LGTM modulo a 🟢 CI.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -90,7 +91,6 @@ requirements:
     # To keep in sync with `install_requires` in setup.cfg.
     - numpy
     - pandas
-    - attrs
```
Comment (jjerphan) [inline recipe/meta.yaml:93]: This is actually required in practice.

It is curious that the test suite does _not_ fail. :thinking: 

<details>

<summary>Traceback of <code>4.3.0rc0 py39hab09b7a_0</code></summary>

```python
In [1]: import arcticdb
---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
Cell In[1], line 1
----> 1 import arcticdb

File ~/.local/share/miniforge3/envs/test/lib/python3.9/site-packages/arcticdb/__init__.py:5
      2 import os as _os
      3 import sys as _sys
----> 5 from arcticdb.arctic import Arctic
      6 from arcticdb.options import LibraryOptions
      7 from arcticdb.version_store.processing import QueryBuilder

File ~/.local/share/miniforge3/envs/test/lib/python3.9/site-packages/arcticdb/arctic.py:14
     12 from arcticdb_ext.storage import LibraryManager
     13 from arcticdb.exceptions import LibraryNotFound, MismatchingLibraryOptions
---> 14 from arcticdb.version_store.library import ArcticInvalidApiUsageException, Library
     15 from arcticdb.version_store._store import NativeVersionStore
     16 from arcticdb.adapters.arctic_library_adapter import ArcticLibraryAdapter

File ~/.local/share/miniforge3/envs/test/lib/python3.9/site-packages/arcticdb/version_store/__init__.py:2
      1 from arcticdb.version_store._common import TimeFrame
----> 2 from arcticdb.version_store._store import NativeVersionStore, VersionedItem

File ~/.local/share/miniforge3/envs/test/lib/python3.9/site-packages/arcticdb/version_store/_store.py:15
     13 import re
     14 import itertools
---> 15 import attr
     16 import warnings
     17 import difflib

ModuleNotFoundError: No module named 'attr'
```

</details>

Comment (jjerphan) [reply recipe/meta.yaml:93]: https://github.com/conda-forge/arcticdb-feedstock/pull/154#issuecomment-1911705930 was opened to reintroduce it.

Comment (jjerphan) [reply recipe/meta.yaml:93]: https://github.com/man-group/ArcticDB/pull/1272 was opened upstream as well.

**Pull request merged**